### DISCUSSION TITLE
How to create a menu from the taxonomy,please?

### DISCUSSION THREAD

**Discussion by effapp** [OrchardCMS/OrchardCore#16577] [Q&A]
How to create a multi-level menu from the taxonomy,please?

**Comment by hishamco** [comment]
Enable the `Taxonomy` feature, then create your taxonomies, after that you loop through each term and get it's children, I remember @deanmarcussen long time ago demo how to use taxonomies in admin menu, right?

**Reply by hishamco** [reply]
Unfortunately there's no code yet in the repo https://github.com/ThisNetWorks/ThisNetWorks.OrchardCore.AdminTree

**Comment by effapp** [comment]
Hi @hishamco ,
I meant to create a multi-level menu from the taxonomy and display it in front-end.

It's the best to automatically produce menu by selecting the taxonomy.

**Reply by hishamco** [reply]
> It's the best to automatically produce menu by selecting the taxonomy.

This could be a custom feature

**Reply by effapp** [reply]
Like Content Menu Item, it produce menu by selecting content item.

**Reply by hishamco** [reply]
Do you mean Admin Menus?

**Reply by effapp** [reply]
no, it's front-end menu.

**Comment by hishamco** [comment]
Is your issue fixed?

**Reply by effapp** [reply]
still unresolved
